git.lua

```diff
@@ -112,6 +112,30 @@ local function branches(token)
     return res
 end
 
+local function alias(token)
+    local res = {}
+
+    -- Try to resolve .git directory location
+    local git_dir = get_git_dir()
+
+    if git_dir == nil then return res end
```

Comment:
`git_dir` is not used anywhere else, so probably these two lines can be combined